qa/suites/fs/multiclient/tasks/fsx-mpi.yaml.disabled

```diff
@@ -1,7 +1,4 @@
 # make sure we get the same MPI version on all hosts
-os_type: ubuntu
-os_version: "14.04"
```

Comment:
should probably just switch this to 16.04.. we still need uniformity?

Reply:
@liewegas i added a link to `../../../../distros/supported/ubuntu_latest.yaml` in this very commit. so i am switching this to 16.04.

Reply:
i see, :+1: